src/components/App.tsx

```diff
@@ -1533,9 +1534,8 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         },
       });
     }
-
     // ensures we don't prevent devTools select-element feature
-    if (event[KEYS.CTRL_OR_CMD] && event.shiftKey && event.key === "C") {
+    if (event[KEYS.CTRL_OR_CMD] && event.shiftKey && event.code === CODES.C) {
```

Comment:
we used `key` here

Reply:
I wanted to avoid having the same key in `KEYS` and `CODES`: https://github.com/excalidraw/excalidraw/pull/2426/files#diff-05a457d6d83e32a543fcc307d09afa44fb1f93ae80c0b12e1b12e86729a28a37L33

Reply:
That's a nice wish, but how does it relate to the actual behavior? :p

Reply:
Everything still work.. Not sure why we even need this line.. we don't use this keyboard shortcut..

Reply:
https://github.com/excalidraw/excalidraw/pull/1446/files

Reply:
Still works as expected without this if..